cli_tools/gce_vm_image_export/exporter/exporter.go

```diff
@@ -101,6 +102,14 @@ func buildDaisyVars(destinationURI string, sourceImage string, format string, ne
 	return varMap
 }
 
+func getFullImagePath(imageName string) string {
+	if !strings.Contains(imageName, "/") {
+		// Extend simple image name to full image name
+		return fmt.Sprintf("global/images/%v", imageName)
```

Comment:
Is this going to work with project specific images?

Reply:
It's "current project image". User can specify any project as current project by --project=aProject.
If the image is from a different project from running project, user has to specify full image path - "projects/another-project/global/images/animage"